css/style.css

```diff
@@ -502,3 +515,330 @@ form {
+    background-repeat: no-repeat;
+    font-family: 'Inter', sans-serif;
+    font-family: 'Poppins', sans-serif;
+
+    /* border: 1px solid yellow; */
+  }
+
+  .container {
+    width: 100%;
+    margin: auto;
+    margin-top: 10px;
+
+    /* border: 10px solid black; */
+
+    /* background-image: url('../images/Frame(1).svg'); */
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: bottom 1px left 20px;
+  }
+
+  .nav-flex {
+    min-width: 100%;
+    margin: auto;
+    height: 1115px;
+  }
+
+  .nav-middle ul {
+    display: inline;
+    display: flex;
+    justify-content: space-evenly;
+  }
+
+  nav {
+    display: flex;
+    min-width: 100%;
+    justify-content: center;
+  }
+
+  nav .nav-left,
+  nav .nav-right {
+    display: none;
+  }
+
+  .nav-middle ul li {
+    margin-right: 25px;
+    font-size: 16px;
+    font-weight: 600;
+    color: #344563;
+    font-family: 'Inter', sans-serif;
+  }
+
+  .home-content {
+    background-image: none;
+    width: 920px;
+    height: 920px;
+    padding-top: 100px;
+    margin: auto;
+  }
+
+  .home-content .primary {
+    height: 114px;
+    width: 920px;
+    margin: 12px;
+    text-align: center;
+  }
+
+  .home-content .supporting {
+    width: 920px;
+    height: 150px;
+    color: #172b4d;
+    font-size: 20px;
+    font-weight: 400;
+    margin-top: 2px;
+    margin-bottom: 0;
+    font-family: 'Inter', sans-serif;
+    padding: 0 60px;
+  }
+
+  .about-me-text h2 {
+    color: #172b4d;
+    font-weight: 700;
+    font-size: 48px;
+    font-style: normal;
+    font-family: 'Inter', sans-serif;
+    margin-bottom: 10px;
+    text-align: center;
+    width: 447px;
+    height: 48px;
+  }
+
+  .home-content .primary h2 {
+    text-align: center;
+    font-size: 40px;
+    line-height: 0.8em;
+    font-weight: 700;
+    color: #172b4d;
+    font-family: 'Inter', sans-serif;
+  }
+
+  .home-content .primary .software-dev {
+    font-size: 48px;
+  }
+
+  .social-icons {
+    margin-top: 0;
+    padding-top: 5px;
+  }
+
+  .my-recent-works {
+    margin-top: 500px;
+    width: 447px;
+    height: 80px;
+    padding-bottom: 0;
+    margin-bottom: 0;
+  }
+
+  .my-recent-works h3 {
+    color: #172b4d;
+    font-size: 30px;
+    font-weight: 700;
+    font-style: normal;
+  }
+
+  .my-recent-works img {
+    display: none;
+  }
+
+  .my-works {
+    min-width: 1171px;
+    min-height: 1020px;
+    margin: auto;
+  }
+
+  .works .work-1,
+  .works .work-2,
+  .works .work-3,
+  .works .work-4,
+  .works .work-5,
+  .works .work-6 {
+    max-width: 369px;
+    max-height: 474px;
+  }
+
+  .works {
+    display: grid;
+    gap: 12px;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  .about-me {
+    width: 100%;
+    height: 345px;
+    margin: auto;
+    background: url("../images/Group\ desktop.svg");
+    background-repeat: no-repeat;
+    background-position: 1098px 0;
+  }
+
+  .about-me-text {
+    width: 100%;
+    height: 345px;
```

Comment:
- [ ] Kindly remove the extra space before the work section and after the about section as per the figma design.
- Your design:
<img width="1119" alt="Screenshot 2022-03-24 at 11 08 48 PM" src="https://user-images.githubusercontent.com/70269507/160006229-5230909d-9e5d-419c-9deb-2c1c5f8d9a71.png">